Makefile

```diff
@@ -514,6 +514,7 @@ test-e2e: test-e2e-setup install-ginkgo
 	-timeout_multiplier=$(E2E_TIMEOUT_MULTIPLIER) \
 	-artifact_dir=$(ARTIFACT_DIR) \
 	-oc_cli=$(OC_CLI) \
+	--ginkgo.vv \
```

Comment:
```
  --vv 
    If set, emits with maximal verbosity - includes skipped and pending tests.
```

Reply:
There are some drawbacks of using `vv`, but I felt is better. Without it, is hard to see in CI logs where a test begins and ends. But I am open for discussion